R/grids.R

```diff
@@ -32,7 +34,7 @@
 #' grid_random(penalty(), mixture())
 #'
```

Comment:
Can you please add an example of `filter` usage?

Reply:
Yes, I have added an example and then ran devtools::document()